instrumentation/apache-httpclient/apache-httpclient-5.0/library/src/main/java/io/opentelemetry/instrumentation/apachehttpclient/v5_0/ApacheHttpClient5HttpAttributesGetter.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.apachehttpclient.v5_0;
+
+import io.opentelemetry.instrumentation.api.semconv.http.HttpClientAttributesGetter;
+import jakarta.annotation.Nullable;
+import java.util.List;
+import org.apache.hc.core5.http.HttpResponse;
+
+enum ApacheHttpClient5HttpAttributesGetter
```

Comment:
why did you prefer what https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/apache-httpclient/apache-httpclient-4.3/library/src/main/java/io/opentelemetry/instrumentation/apachehttpclient/v4_3/ApacheHttpClientHttpAttributesGetter.java has to what https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/apache-httpclient/apache-httpclient-5.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/apachehttpclient/v5_0/ApacheHttpClientHttpAttributesGetter.java has?

Reply:
I changed now basing my class in the 5.0 class.